### DISCUSSION TITLE
Size difference between universal and non-universal self-extracting ZIP files

### DISCUSSION THREAD

**Discussion by pivic** [gildas-lormeau/SingleFile#1253] [Q&A]
Thanks a million for the new self-extracting ZIP files feature!

The new features page states that non-universal files are about 1% smaller than universal files. However, the examples included [differ around 33% in size](https://hyp.is/5AAbLmNHEe6ckvuxi8gVFQ/gildas-lormeau.github.io/singlefile-updates/version-1-22.html).

Is the example just a general estimation on size difference or is something wrong?

**Comment by gildas-lormeau** [comment]
Actually, the examples show the difference between a (universal) self-extracting page and a classic HTML file. Self-extracting files should indeed be at least 30% smaller than HTML files.
